src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -398,17 +399,22 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
     }
 
     private fun displayEmptyDescription() {
-        if (isScanRunning(project)) {
-            displayScanningMessage()
-        } else if (rootOssTreeNode.childCount == 0
-            && rootSecurityIssuesTreeNode.childCount == 0
-            && rootQualityIssuesTreeNode.childCount == 0
-        ) {
-            displayNoVulnerabilitiesMessage()
-        } else {
-            displaySelectVulnerabilityMessage()
+        when {
+            isScanRunning(project) -> {
+                displayScanningMessage()
+            }
+            isCliDownloading() -> {
+                displayDownloadMessage()
+            }
+            rootOssTreeNode.childCount == 0
+                && rootSecurityIssuesTreeNode.childCount == 0
```

Comment:
since it's a new detekt issue, can we have it fixed? [official Kotlin example](https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements)